src/config.cc

```diff
@@ -151,9 +151,6 @@ void Config::postUpdateValues() {
   if (uptane.director_server.empty()) uptane.director_server = tls.server + "/director";
 
   if (uptane.ostree_server.empty()) uptane.ostree_server = tls.server + "/treehub";
-
-  uptane.public_key_path = (tls.certificates_directory / uptane.public_key_path).string();
-  uptane.private_key_path = (tls.certificates_directory / uptane.private_key_path).string();
```

Comment:
I also find it pleasant that postUpdateValues which was previously doing a lot of things is now just performs a simple task of reading the server URL from an archive and setting a few variables accordingly.